elements/lisk-dpos/test/unit/apply.spec.ts

```diff
@@ -54,6 +59,7 @@ describe('dpos.apply()', () => {
 	let stateStore: StateStoreMock;
 	const randomSeed1 = Buffer.from('283f543e68fea3c08e976ef66acd3586');
 	const randomSeed2 = Buffer.from('354c87fa7674a8061920b9daafce92af');
+	const initDelegates = delegatePublicKeys.map(pk => Buffer.from(pk, 'hex'));
```

Comment:
Would it be not more consistent to use addresses as identifiers? If you have a block with generatorPublicKey property, it is super fast to compute the address from it.

Reply:
We changed to addresses as identifiers a few month backs, also the shuffling and selection of standby delegates uses addresses and not public keys any more. Note that public keys will also be removed from accounts.

Reply:
I agree we should change it to the address, but currently genesis block holds the public key, so it will handle in the removing address issue #5471

Reply:
My misunderstanding, actually this should be already address, will be fix in this PR